store/common/fields.py

```diff
@@ -0,0 +1,41 @@
+# store/common/fields.py
+from decimal import Decimal, InvalidOperation
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.db import models
+from rest_framework import serializers
+
+
+class PriceField(models.IntegerField):
+
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault('validators', [
+            MinValueValidator(1),
+            MaxValueValidator(4800000)
```

Comment:
It seems max value here is higher than in line 27 - I would expect both to be the same. 

Maybe it's best to have a variable definition for max value in this file.

Then, use the variable in both places.